src/pages/marketplace.tsx

```diff
@@ -10,15 +11,17 @@ class Marketplace extends React.PureComponent {
     public render = () => {
         return (
             <>
-                <Sidebar>
-                    <BuySellContainer />
-                </Sidebar>
-                <MainContent>
-                    <OrderHistoryContainer />
-                </MainContent>
-                <Sidebar>
-                    <OrderBookTableContainer />
-                </Sidebar>
+                <AdBlockDetector>
```

Comment:
Hmm, I don't see why this should wrap the other components; just having it "beside" them should be enough. @unjapones what do you think?

Reply:
Yup, no need to wrap any content here. We can probably add it to [`/src/index.tsx`](https://github.com/0xProject/0x-launch-kit-frontend/blob/development/src/index.tsx#L25) as a sibling of `GeneralLayout`, i.e.:
```jsx
            <AppContainer>
                <AdBlockDetector/>
                <GeneralLayout>
                    <Switch>

```

Reply:
Done, thanks!